### PR TITLE
Update CRD subresources doc for 1.11

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/kubernetes/pull/64766

/hold
until the above PR is merged

/assign sttts mistyhacks

### CODE REVIEW

File: content/en/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md [hunk 1]
```diff
@@ -354,7 +354,7 @@ When the status subresource is enabled, the `/status` subresource for the custom
 - `PUT` requests to the `/status` subresource only validate the status stanza of the custom resource.
 - `PUT`/`POST`/`PATCH` requests to the custom resource ignore changes to the status stanza.
 - Any changes to the spec stanza increments the value at `.metadata.generation`.
-- `Properties` and `Required` are the only constructs allowed in the root of the CRD OpenAPI validation schema.
+- `Properties`, `Required` and `Description` are the only constructs allowed in the root of the CRD OpenAPI validation schema.
```
Comment (sttts) [inline content/en/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md]: maybe lower-case

Comment (nikhita) [reply content/en/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md]: Done.

**Comment by nikhita** [comment]
https://github.com/kubernetes/kubernetes/pull/64766 merged. Removing hold.
/hold cancel

**Comment by nikhita** [comment]
Can someone add this to the milestone to make sure this is not missed?

**Comment by nikhita** [comment]
@enisoc thanks!

**Comment by zacharysarah** [comment]
@nikhita This PR looks good, but @mistyhacks needs to approve it because she's the release meister for the 1.11 milestone.

@mistyhacks I verified the preview looks good. 👍

**Comment by mistyhacks** [comment]
Thanks all for waiting for me.

/approve

**Pull request merged**